test/parallel/test-module-create-require.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const path = require('path');
+const { URL } = require('url');
```

Comment:
fwiw, `URL` should be a global now. There's no reason to require it.

Reply:
are there any release lines where that hasn't been backported yet?

Reply:
Yes, it's not on v8.x: https://github.com/nodejs/node/pull/20306